sda/cmd/ingest/ingest.go

```diff
@@ -388,6 +390,20 @@ func main() {
 							continue mainWorkLoop
 						}
 
+						// Set the file's hex encoded public key
+						log.Debugln("Compute and set key hash")
```

Comment:
Will these get ignored when deployed? I don't remember the loglevel we set. Just wondering if they add any value or if they will flood our logs.

Reply:
Well spotted. I don't know about the log level for the deployed version, but I removed the line anyway.

Reply:
For production we run at log level `info`, for staging normally `debug` in order to easier be able to see what's happening if things goes wrong.